src/System.Data.Common/src/System/Data/IDataReader.cs

```diff
@@ -0,0 +1,34 @@
+//------------------------------------------------------------------------------
+// <copyright file="IDataReader.cs" company="Microsoft">
+//      Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// <owner current="true" primary="true">markash</owner>
+// <owner current="true" primary="false">laled</owner>
+//------------------------------------------------------------------------------
+
+namespace System.Data {
```

Comment:
Nit: the coding style for corefx is Allman style braces, where the braces start on a new line.